build.gradle

```diff
@@ -233,6 +233,7 @@ dependencies {
 
   compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.37'
   compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.37'
+  compile group: 'commons-httpclient', name: 'commons-httpclient', version: '3.1'
```

Comment:
ummm, See here, please: https://tools.hmcts.net/confluence/display/WA/WA+Dev+Team+Guide

[We prefer to use Feign Client for Rest calls but will also use restTemplate when appropriate.]

Also, this library you use is tagged as "end of life" see here: https://hc.apache.org/httpclient-3.x/ 

I don't think it's a good idea to use it. 

You could just use the new apache common library https://hc.apache.org/httpcomponents-client-ga/, or restTemplate which is part of spring-boot or feign where we have examples in other projects or just the new java 11 HTTPClient: 
http://openjdk.java.net/groups/net/httpclient/intro.html